src/zeit/content/article/breaking.py

```diff
@@ -37,3 +40,26 @@ def banner_matches(self, banner):
 @grok.implementer(zeit.connector.interfaces.IWebDAVProperties)
 def webdav_properties(context):
     return zeit.connector.interfaces.IWebDAVProperties(context.context)
+
+
+class BreakingNewsArticle(zeit.content.article.article.Article):
+    """Article is the main content type in the Zeit CMS."""
+
+    zope.interface.implements(zeit.content.article.interfaces.IBreakingNews,
+                              zeit.cms.interfaces.IEditorialContent)
+
+    zeit.cms.content.dav.mapProperties(
+        zeit.content.article.interfaces.IBreakingNews,
+        zeit.cms.interfaces.DOCUMENT_SCHEMA_NS,
+        ('is_breaking',))
+
+    def banner_matches(self, banner):
+        return IBreakingNewsBody(banner).article_id == self.uniqueId
+
+
+class BreakingNewsArticleType(zeit.cms.type.XMLContentTypeDeclaration):
```

Comment:
Da war ich mich in der intefaces.py noch nicht ganz sicher, was da noch gebraucht wird. Habe es jetzt entfernt. Danke.